README.md

```diff
@@ -157,7 +157,7 @@ In order to use the API within `polkadot.js` you'll need to configure the follow
     "_enum": ["Disabled", "Enabled"]
   },
   "Restriction": {
-    "_enum": ["None", "SenderOwnsAllInputs"]
+    "_enum": ["None", "SenderOwnsAllInputs", "FixedNumberOfInputs"]
```

Comment:
might need a <u32> on the end? I'm not 100% sure

Reply:
I've updated the readme from what I've seen in the docs so should hopefully be right